content/en/blog/_posts/2022-12-06-non-graceful-node-shutdown-to-beta.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: blog
+title: "Kubernetes 1.26: Non-Graceful Node Shutdown Moves to Beta"
+date: 2022-12-06T10:00:00-08:00
+slug: kubernetes-1-26-non-graceful-node-shutdown-beta
+---
+
+**Author:** Xing Yang and Ashutosh Kumar (VMware)
+
+Kubernetes v1.24 introduces [alpha support](https://kubernetes.io/blog/2022/05/20/kubernetes-1-24-non-graceful-node-shutdown-alpha/) for [Non-Graceful Node Shutdown](https://github.com/kubernetes/enhancements/tree/master/keps/sig-storage/2268-non-graceful-shutdown).
+In Kubernetes v1.26, this feature moves to beta. This feature allows stateful workloads to failover to a different node after the original node is shutdown or in a non-recoverable state such as the hardware failure or broken OS.
+
+## What is a Node Shutdown in Kubernetes
+
+In a Kubernetes cluster, it is possible for a node to shutdown. This could happen either in a planned way or it could happen unexpectedly. You may plan for a security patch, a kernel upgrade, and need to reboot the node; or a node may shutdown due to preemption of VM instances; or a node may shutdown due to a hardware failure, or a software problem, etc.
```

Comment:
```suggestion
In a Kubernetes cluster, it is possible for a node to shut down. This could happen either in a planned way or it could happen unexpectedly. You may plan for a security patch, or a kernel upgrade and need to reboot the node, or it may shut down due to preemption of VM instances. A node may also shut down due to a hardware failure or a software problem.
```

Reply:
Done